benchmarks/README.md

```diff
@@ -0,0 +1,188 @@
+# Description
+
+This folder contains benchmark running scenarios and their resulting outputs continuously. If you want to execute them, you can do so by using the following command:
```

Comment:
This first sentence does not make any sense, please revise.

Reply:
Also: "These can be executed by using the following command", sounds more professional.